Added ability to resize badge icons.

targeter21 opened the pull request:
PR to address issue #40 by making the icon width and height configurable through CSS variables.

targeter21 commented:
I ran wct locally and there were no errors; looks like Travis broke for a different reason and I'm unclear on how to fix it.

targeter21 commented:
I was able to fix it by updating the Firefox version Travis uses to 48, the latest.

azakus commented:
Firefox 48+ is using the new [geckodriver](https://github.com/mozilla/geckodriver) selenium driver, which we have been seeing issues with flaky tests in the past.

@rictic, we should evaluate if 48 is ready for primetime with geckodriver.

rictic commented:
+1 to reevaluating 48. I'd be down for that on Monday.

notwaldorf commented:
@targeter21 can you undo your Travis change, please? It's fine if the Firefox tests are red, it won't affect me merging the PR

notwaldorf commented on paper-badge.html:
```diff
@@ -78,8 +78,8 @@
       iron-icon {
-        --iron-icon-width: 12px;
-        --iron-icon-height: 12px;
+        --iron-icon-width: var(--badge-icon-width, 12px);
```
Can you please document these properties in the `Styling` section of the docs?

targeter21 replied:
Documentation added

notwaldorf commented:
@targeter21 Does this PR fix https://github.com/PolymerElements/paper-badge/issues/40? If so, can you add that to your description, please?

notwaldorf commented on demo/index.html:
```diff
@@ -107,6 +107,36 @@
+        </paper-badge>
+
+        <style is="custom-style">
+          paper-badge {
```
i think this style is actually global, so you might want to do something like `paper-badge.large` instead

targeter21 commented:
Updated the descriptions, added documentation, and changed the variables to better align with the other variable names. Also removed the Travis CI commit.

targeter21 commented on paper-badge.html:
```diff
@@ -55,6 +55,8 @@
 `--paper-badge-width` | The width of the badge circle | `20px`
 `--paper-badge-height` | The height of the badge circle | `20px`
+`--paper-badge-icon-width` | The width of the badge icon, allowed up to the width of the badge | `12px`
+`--paper-badge-icon-height` | The height of the badge icon, allowed up to the width of the badge | `12px`
```
Small typo. Need to change width to height.

notwaldorf commented:
Would it be possible to add a unit test, please, to make sure this doesn't break in the future?

targeter21 commented:
Added a wct test to verify that the iron-icon dimensions match the CSS configuration when given. The array index access seems a bit hokey, not sure if there is a better way to accomplish that.

targeter21 commented:
Going to go ahead and close this. If this is still a problem the code can probably be ported to the current master.

knyto2 commented:
Hi,

Any updates on this? I would love to get this code in so that it would resolve my issue of being able to resize the iron-icon.

targeter21 commented:
Hi @knyto2, right now the branch I believe had conflicts and due to inactivity, I closed it. You likely can port it to the current master in a new PR.
